index.js

```diff
@@ -212,7 +220,15 @@ NYC.prototype._maybeInstrumentSource = function (code, filename, relFile) {
     return null
   }
 
-  return this.transform(code, {filename: filename, relFile: relFile})
+  var ext, transform
+  for (ext in this.transforms) {
```

Comment:
I was thinking you could iterate over `this.extensions` here, though we're veering quite far into premature optimization land.

Reply:
why not making a ext => transform object/Map?